### ISSUE TITLE
Code block tools: output from `tombi` doesn't provide enough context

### ISSUE THREAD

**Issue by eread** [rvben/rumdl#506]
### rumdl version

0.1.43

### Installation method

mise

### Bug description

`tombi` (https://tombi-toml.github.io/tombi/) is a TOML linter and formatter. `rumdl` includes a definition for `tombi` to make it simpler to set up.

However, when `tombi` encounters a problem, only the final line of the output is returned to the user, so much valuable detail is lost.

### Steps to reproduce

1. Create a Markdown file with this content (the TOML is invalid):
   ````markdown
   ---
   title: Heading
   ---
   
   Some introductory text.
   
   ```toml
   [test]
   test: "test"
   ```
   ````
1. Create a `.rumdl.toml` file with the following content:
   ```toml
   [code-block-tools]
   enabled = true

   [code-block-tools.languages]
   toml = { lint = ["tombi"] }
   ```
1. For comparison, create a TOML file with the following content (the TOML is invalid):
   ```toml
   [test]
   test: "test"
   ```
1. Run `rumdl check` on the Markdown file.
1. Run `tombi lint` on the TOML file. 

### Expected behavior

The output of `rumdl check` would be as detailed as the output of `tombi lint`.

### Actual behavior

`rumdl check` returns only:

```shell
test.md:7:1: [tombi] 1 file failed to be formatted
```

This is the final line of a call to `tombi format` (not `tombi lint`, incidentally) and exact errors are missing.

### Configuration

```toml
[code-block-tools]
enabled = true

[code-block-tools.languages]
toml = { lint = ["tombi"] }
```

### Operating system

macOS

**Comment by rvben** [comment]
Fixed in v0.1.44. Code block tool diagnostics now include all output lines from the tool, providing full context instead of just the last line. Thanks for reporting @eread!

**Comment by eread** [comment]
> Fixed in v0.1.44. Code block tool diagnostics now include all output lines from the tool, providing full context instead of just the last line. Thanks for reporting [@eread](https://github.com/eread)!

@rvben Could you double check this one? If I'm not mistaken, I still have the issue in version `0.1.45`. I've run `rumdl clean` to make sure my cache was clean.

**Comment by rvben** [comment]
Fixed in v0.1.46. Two changes: (1) stdout and stderr are now combined and multi-line error formats are properly parsed, so full diagnostic context is shown. (2) The default `tombi` tool definition now runs `tombi lint` instead of `tombi format` when configured in the `lint` slot — use `tombi:format` explicitly for formatting. Thanks for reporting @eread!
